core/bin/external_node/src/metrics/mod.rs

```diff
@@ -31,13 +31,13 @@ pub(crate) struct ExternalNodeMetrics {
 impl ExternalNodeMetrics {
     pub(crate) fn observe_config(
         &self,
-        l1_chain_id: L1ChainId,
+        sl_chain_id: SLChainId,
         l2_chain_id: L2ChainId,
         postgres_pool_size: u32,
     ) {
         let info = ExternalNodeInfo {
             server_version: SERVER_VERSION,
-            l1_chain_id: l1_chain_id.0,
+            l1_chain_id: sl_chain_id.0,
```

Comment:
Will L1 chain ID stay as an EN configuration parameter? If so, IMO it'd make more sense to *add* a new field to `ExternalNodeInfo` rather than redefining semantics of an existing field.